rclcpp/test/rclcpp/CMakeLists.txt

```diff
@@ -262,6 +267,11 @@ ament_add_gtest(test_node_interfaces__node_waitables
 if(TARGET test_node_interfaces__node_waitables)
   target_link_libraries(test_node_interfaces__node_waitables ${PROJECT_NAME} mimick)
 endif()
+ament_add_gtest(test_node_interfaces__test_template_utils
+  node_interfaces/detail/test_template_utils.cpp)
```

Comment:
Since this file isn't actually a gtest, is this ok? I'm actually not sure if it is or not. gtest command-line arguments, for example, will not work with this executable.

Maybe instead this should just be a target (not a gtest target) or maybe the `test_template_utils.cpp` should have the skeleton of a gtest main function and stuff in it.